File: src/main/java/yaasl/server/security/SecurityConstants.java
package yaasl.server.security;

public class SecurityConstants {

    public static final long EXPIRATION_TIME = 10 * 60 * 60 * 1000;
    public static final int REMEMBER_ME_EXPIRATION_TIME = 10 * 24 * 60 * 60;
    public static final String TOKEN_PREFIX = "Bearer ";
    public static final String TOKEN_HEADER = "Authorization";
    public static final String WEB_SOCKET_TOKEN_HEADER = "Sec-WebSocket-Protocol";
    public static final String SIGN_UP_URL = "/login";
    public static final String REMEMBER_ME_COOKIE = "yaasl-remember-me";

}

File: src/main/kotlin/yaasl/server/security/SecurityConstants.kt
package yaasl.server.security

object SecurityConstants {

    val EXPIRATION_TIME = (10 * 60 * 60 * 1000).toLong()
    val REMEMBER_ME_EXPIRATION_TIME = 10 * 24 * 60 * 60
    val TOKEN_PREFIX = "Bearer "
    val SIGN_UP_URL = "/login"
    val REMEMBER_ME_COOKIE = "yaasl-remember-me"

}
